### DISCUSSION TITLE
Armoury Create Disabled Win Key but I get Service Error when launching it, any way to reenable using GHelper?

### DISCUSSION THREAD

**Discussion by jankeith** [seerge/g-helper#2477] [Q&A]
Armoury Create Disabled Win Key but I get Service Error when launching it, any way to reenable using GHelper?

**Comment by seerge** [comment]
@jankeith try [hardware reset](https://github.com/seerge/g-helper/wiki/Troubleshooting#hardware-reset--cmos-reset) - it will probably reset all settings done by AC previously
